src/libs/antares/benchmarking/studyinfo.h

```diff
@@ -1,33 +1,40 @@
 #pragma once
 
-namespace Antares {
-  namespace Data {
-    class Study;
-  }
-}
+#include <antares/study.h>
 
 namespace Benchmarking
 {
-struct StudyInfo
+class StudyInfo
 {
-    StudyInfo();
-    void initializeFromStudy(const Antares::Data::Study&);
-    // Business problem
-    unsigned int nbEnabledYears;
-    unsigned int nbEnabledClusters;
-    unsigned int nbEnabledBindingConstraints;
-    unsigned int nbAreas;
-    unsigned int nbLinks;
-    unsigned int UnitComitmentMode;
+public:   
+    StudyInfo(const Antares::Data::Study& study);
+    void collect();
+    void flush(Yuni::String& filePath);
+
+private:
+    // Methods
+    void getNbPerformedYears();
+    void getEnabledThermalClusters();
```

Comment:
I suggest passing `study` to each method, not to the class constructor.
```suggestion
    StudyInfo();
    void collect(const Antares::Data::Study& study);
    void flush(Yuni::String& filePath);

private:
    // Methods
    void getNbPerformedYears(const Antares::Data::Study& study);
    void getEnabledThermalClusters(const Antares::Data::Study& study);
```